### PR TITLE
Add completed-notebooks and rename virtual-setup

### LINKED ISSUE #121: Add `completed-notebooks` and rename `virtual-setup`
_Broken out of #109_

This issue should specifically cover the following changes originally recorded in #109:

- Create new directory `completed-notebooks` with a `.gitkeep` file
- Rename directory `virtual-setup` -> `software-setup`, and accordingly update relative links throughout repo

### PR DESCRIPTION
**Pull request by sjspielman**
**⚠️ Stacked on #127**
Closes #121 

This PR accomplishes three main things:

1. First a small thing as a review treat: create `completed-notebooks/.gitkeep`
2. Rename `virtual-setup` -> `software-setup`. Kind of a treat!
3. Update relative links throughout the repo to accommodate this rename. This one is much less of a treat but hopefully not too bad! At this point, `grep` for me only returns two files where "virtual-setup" is still mentioned - the README which will be updated towards the end of all of this, and setup issue template that will be dealt with as part of #124.

### CODE REVIEW

**Comment by sjspielman** [comment]
Ah woops @jashapiro didn't mean to click request on a Friday afternoon here! Not expecting a review here until you're back from PTO. Note that I think you're the right person to review this one, because `grep`.

File: software-setup/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Setup for Data Lab Virtual Workshops
+title: Setup Software for Data Lab Workshops
```
Comment (jashapiro) [inline software-setup/README.md:2]:
```suggestion
title: Set Up Software for Data Lab Workshops
```

**Review by jashapiro — APPROVED** [review]
LGTM, though I didn't check that it all renders. I'm saving that for later reviews.

**Comment by sjspielman** [comment]
thanks for having a look!

**Pull request merged**